### PR TITLE
Remove left over print statments

### PR DESCRIPTION
**Pull request by eslickj**
## Fixes

It's a good thing I'm not a surgeon.  I left behind a couple print() statements. 

### Legal Acknowledgement

By contributing to this software project, I agree to the following terms and conditions for my contribution:

1. I agree my contributions are submitted under the license terms described in the LICENSE.txt file at the top level of this directory.
2. I represent I am authorized to make the contributions and grant the license. If my employer has rights to intellectual property that includes these contributions, I represent that I have received permission to make contributions and grant the required license on behalf of that employer.

### CODE REVIEW

**Review by dallan-keylogic — DISMISSED** [review]

File: idaes/core/util/model_serializer.py [hunk 1]
```diff
@@ -735,7 +733,7 @@ def to_json(
     # unfortunatly I can't write how long it took to write the file in the file
     pdict["etime_write_file"] = file_time - dict_time
     if return_dict:
-        # In interactive environments returning the dict can cuase it to print
+        # In interactive environments returning the dict can cause it to print
         # an extreemly large amount of stuff.  So added this option to make sure
```
Comment (andrewlee94) [inline idaes/core/util/model_serializer.py:737]: Typo: extremely

**Review by andrewlee94 — APPROVED** [review]
To make things worse, in reviewing this PR I found a typo as well.

Otherwise looks good.

File: idaes/core/util/model_serializer.py (hunk 1 above)
Comment (eslickj) [reply idaes/core/util/model_serializer.py:737]: Aw dang.  I need to look for a comment/string spell checker

**Review by dallan-keylogic — APPROVED** [review]

**Review by andrewlee94 — APPROVED** [review]
Let me know if you find something that can spell check code - I need it myself.

**Pull request merged**